prow/istio-presubmit.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ex
+
+if [ "${CI}" == "bootstrap" ]; then
+    # ensure correct path
+    mkdir -p ${GOPATH}/src/istio.io
+    mv ${GOPATH}/src/github.com/istio/istio ${GOPATH}/src/istio.io
+    cd ${GOPATH}/src/istio.io/istio/
+fi
+
+echo 'Running Linters'
+./bin/linters.sh
+
+echo 'Running Unit Tests'
+bazel test //...
+
+echo 'Running Integration Tests'
+./tests/e2e.sh --test_logs_path='_artifacts'
```

Comment:
I believe this should be `--test_log_path`, note no plural. based on the build log [here](https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/464/istio-presubmit/3/)